static_src/components/service_instance.jsx

```diff
@@ -186,7 +185,8 @@ export default class ServiceInstance extends React.Component {
               { serviceInstance.name }
             </span>
           </span>
-          <span className={ this.styler('panel-column') }>
+          <span className={ this.styler('panel-column', 'panel-column-less',
```

Comment:
This sucks to have to use `last` classes, because then you have to change them if the content changes. What are the new styles that require this?